edge-modules/TestAnalyzer/Program.cs

```diff
@@ -32,23 +34,42 @@ static async Task Main(string[] args)
             await CreateWebHostBuilder(args).Build().RunAsync(cts.Token);
         }
 
+        static async Task<DateTime> LoadStartupTimeFromStorage()
+        {
+            DateTime lastReceivedAt = DateTime.MinValue;
+            await ReportingCacheWithStorage.Instance.Init(Settings.Current.StoragePath, Settings.Current.OptimizeForPerformance);
+            var messagesSnapShot = ReportingCache.Instance.GetMessagesSnapshot();
+            foreach (KeyValuePair<string, IList<SortedSet<MessageDetails>>> moduleMesssages in messagesSnapShot)
```

Comment:
do we still use SortedSet?

Reply:
A lot of the analyzer diff is showing when I did not change anything (due to renamed files). I did not touch this code block, so yes we still use.